lpa-submissions-e2e-tests/wait.js

```diff
@@ -0,0 +1,29 @@
+const fetch = require("node-fetch");
+
+console.log('Waiting for ZAP');
```

Comment:
An omission perhaps?

Reply:
Thanks I have removed this now - can you re-review?